lib/jsduck/options/input_files.rb

```diff
@@ -33,7 +33,7 @@ def expand_filename(fname)
 
         if File.exists?(fname)
           if File.directory?(fname)
-            Dir[fname+"/**/*.{js,scss}"].each {|f| files << f }
+            Dir.glob([fname+"/**{,/*/**}/*.{js,scss}"],0).each {|f| files << f }
```

Comment:
Thanks for the patch, but I don't quite understand this code.

- What's the significance of the `0` for flags parameter? To my understanding it's the same as passing no flags.
- Why are we calling Dir.glob() with an array of one string, instead of just the string itself?
- What's the trick with `{,/*/**}` ?

Nothing in [Dir.glob() docs](http://ruby-doc.org/core-2.2.0/Dir.html#method-c-glob) really explained this to me.